src/components/ExerciseZeroForm.js

```diff
@@ -39,19 +36,18 @@ const options = [
 ];
 
 // Here comes the JSX part
-// The form is a stateless component in which we just plug the onSubmit method
+// The form is a stateless component in which we just plug the onSubmit (handleSubmit) method
```

Comment:
Just added the extra reference here about `onSubmit` but called `handleSubmit` in the component.